lib/remote_docker.ex

```diff
@@ -0,0 +1,16 @@
+defmodule RemoteDockers do
+  @moduledoc """
+  Elixir wrapper to drive Docker nodes.
+
+  Before try this library, be sure your docker machine will serve the API on HTTP (on some OS, default is on a socket).
```

Comment:
_Before using this library, ensure your Docker instance is serving the HTTP API (by default, the service is generally listening to a socket)_